tests/test_code_report.py

```diff
@@ -23,13 +23,20 @@ def __init__(self):
         self.text = "from universum.configuration_support import Configuration, Step\n"
         self.text += "configs = Configuration()\n"
 
-    def add_analyzer(self, analyzer: str, arguments: List[str], extra_cfg: str = '') -> 'ConfigData':
-        args = [f", '{arg}'" for arg in arguments]
-        cmd = f"['{python()}', '-m', 'universum.analyzers.{analyzer}'{''.join(args)}]"
+    def add_cmd(self, name: str, cmd: str, step_cfg: str = '') -> 'ConfigData':
+        step_cfg = ', ' + step_cfg if step_cfg else ''
         self.text +=\
-            f"configs += Configuration([Step(name='Run {analyzer}', {extra_cfg} code_report=True, command={cmd})])\n"
+            f"configs += Configuration([Step(name='{name}', command={cmd}{step_cfg})])\n"
         return self
 
+    def add_analyzer(self, analyzer: str, arguments: List[str], step_cfg: str = '') -> 'ConfigData':
+        name = f"Run {analyzer}"
+        args = [f", '{arg}'" for arg in arguments]
+        cmd = f"['{python()}', '-m', 'universum.analyzers.{analyzer}'{''.join(args)}]"
+        step_cfg = ', ' + step_cfg if step_cfg else ''
```

Comment:
Is there a reason we're using `cfg` instead of `config` as in other modules like `Launcher`?

Reply:
I have no preference either way, I'll change it to 'config' then. But please note that this is a test case code.